kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx

```diff
@@ -378,8 +376,8 @@ class SnapshotSchedule extends Component {
           err="Failed to get snapshot schedule settings"
           loading={loadingConfig}
         />
-        {!isAppConfig && isKurlEnabled &&
-          <GettingStartedSnapshots isSettingsPage={true} />
+        {!isAppConfig && !isSettingsPage &&
+          <GettingStartedSnapshots isVeleroInstalled={true} />
```

Comment:
are you sure?

Reply:
yep, `isSettingsPage` isn't used inside of GettingStartedSnapshots.  At any rate, no modal should be blocking settings pages.